pkg/services/alerting/notifiers/googlechat.go

```diff
@@ -126,13 +126,15 @@ func (gcn *GoogleChatNotifier) Notify(evalContext *alerting.EvalContext) error {
 		gcn.log.Error("evalContext returned an invalid rule URL")
 	}
 
-	// add a text paragraph widget for the message
-	widgets := []widget{
-		textParagraphWidget{
+	var widgets []widget
```

Comment:
Need to make sure it's initialized, will fail further down below otherwise.

```suggestion
	widgets := []widget{}
```